docs/docs/examples/vector_stores/DatabricksVectorSearchDemo.ipynb

```diff
@@ -192,6 +204,99 @@
     "\n",
     "print(response.response)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Advanced Usage (Auto Merging Retriever)\n",
+    "\n",
+    "This feature is enabled by the addition of node relationships to the index.\n",
+    "\n",
+    "Source: https://docs.llamaindex.ai/en/stable/examples/retrievers/auto_merging_retriever/"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Initialize the node parser\n",
+    "node_parser = HierarchicalNodeParser.from_defaults()\n",
+    "\n",
+    "# Split documents into nodes and save leaf nodes\n",
+    "nodes = node_parser.get_nodes_from_documents(documents)\n",
+    "leaf_nodes = get_leaf_nodes(nodes)\n",
+    "print(len(leaf_nodes))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create a vector search index\n",
+    "# it must be placed inside a Unity Catalog-enabled schema\n",
+    "\n",
+    "# We'll use self-managed embeddings (i.e. managed by LlamaIndex) rather than a Databricks-managed index\n",
+    "databricks_index = client.create_direct_access_index(\n",
+    "    endpoint_name=\"llamaindex_dbx_vector_store_test_endpoint\",\n",
+    "    index_name=\"my_catalog.my_schema.my_test_table\",\n",
+    "    primary_key=\"my_primary_key_name\",\n",
+    "    embedding_dimension=1536,  # match the embeddings model dimension you're going to use\n",
+    "    embedding_vector_column=\"my_embedding_vector_column_name\",  # you name this anything you want - it'll be picked up by the LlamaIndex class\n",
+    "    schema={\n",
+    "        \"my_primary_key_name\": \"string\",\n",
+    "        \"my_embedding_vector_column_name\": \"array<double>\",\n",
+    "        \"text\": \"string\",  # one column must match the text_column in the DatabricksVectorSearch instance created below; this will hold the raw node text,\n",
+    "        \"doc_id\": \"string\",  # one column must contain the reference document ID (this will be populated by LlamaIndex automatically)\n",
+    "        # add any other metadata you may have in your nodes (Databricks Vector Search supports metadata filtering)\n",
+    "        # NOTE THAT THESE FIELDS MUST BE ADDED EXPLICITLY TO BE USED FOR METADATA FILTERING\n",
```

Comment:
Does this comment apply to `node_info` field? IIUC `node_info` is not meant to be used for metadata filtering. Also let's clearly mention the field name `node_info` cannot be changed as it is hard-coded in our VS class.